src/Client.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace PagarMe;
```

Comment:
Que acham de colocar o cliente dentro de um namespace mais específico? Minha sugestão seria colocar dentro de `PagarMe\Http` por exemplo. O mesmo serviria pra `RequestHandler` e `ResponseHandler`

Reply:
Não sei se isso entrega algum valor para o uso final. Explico:

Teríamos algo assim:

```php
$client = new \PagarMe\Http\Client('mysecretapikey');
```

Do que algo asim:

```php
$client = new \PagarMe\Client('mysecretapikey');
```

Honestamente eu não vejo diferenças aqui, além de uma namespace mais longa.

Reply:
Entendi. Sobre os pontos que comentou:

> Ao ler o trecho acima eu consigo inferir que: é uma classe Client responsável por requisições Http.

Entendo que assim a visibilidade funciona bem, mas não vejo o que isso agrega para o cliente final. IMHO, para o consumidor final (desenvolvedor implementando a API no código), se a classe faz requests por HTTP ou por outro meio, isso pode não ser relevante. O desejo é de apenas integrar o código à API do Pagar.me, seja por qual meio for possível. Novamente, esse é o meu ponto de vista :wink: 

> Quando se separa por domínio temos outro ganho: Dado que `Client` pertence à `Http` fica muito fácil saber que, numa evolução um `Client` responsável por `Ssh` por exemplo são implementações distintas e evitaria que `SRP` fosse ferido. Que acha?

Concordo na vantagem da separação de conceitos, mas acho que neste projeto isso não se aplica. Sabemos que, neste projeto, teremos somente o SDK para integração via HTTP e que outros protocolos não serão abordados. Logo, pensar em usar a separação como meio de organizar entre diferentes protocolos acaba saindo como otimização prematura. Faz sentido?